rule/method/src/main/java/io/opentracing/contrib/specialagent/rule/method/MethodAgentIntercept.java

```diff
@@ -0,0 +1,90 @@
+package io.opentracing.contrib.specialagent.rule.method;
+
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Queue;
+
+/**
+ * MethodAgentIntercept
+ *
+ * @author code98@163.com
+ * @date 2019/11/15 5:28 下午
+ */
+public class MethodAgentIntercept {
+
+    private static final ThreadLocal<Queue<Span>> spanHolder = ThreadLocal.withInitial(LinkedList::new);
```

Comment:
@code98, please change this to jvm1.7 compliant code. Also, avoid using `LinkedList`, because its performance is much worse than `ArrayList`.